org.okstar.ok-msg.yml

```diff
@@ -0,0 +1,97 @@
+id: org.okstar.ok-msg
+runtime: org.kde.Platform
+runtime-version: '5.15-23.08'
+sdk: org.kde.Sdk
+command: ok-msg-desktop
+rename-icon: ok-msg
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --socket=pulseaudio
+  - --device=all
+  - --talk-name=org.gnome.Mutter.IdleMonitor
+  - --talk-name=org.kde.StatusNotifierWatcher
+  - --talk-name=com.canonical.AppMenu.Registrar
+  - --talk-name=com.canonical.indicator.application
+
+add-extensions:
+  org.freedesktop.Platform.ffmpeg-full:
+    directory: lib/ffmpeg
+    add-ld-path: .
+    version: '23.08'
+    autodownload: true
+    autodelete: false
+
+build-options:
+  strip: false
+  no-debuginfo: true    
+  build-args:
+    - --share=network
+cleanup:
+  - /doc
+  - /include
+  - /lib/cmake
+  - /lib/libexec
+  - /lib/metatypes
+  - /lib/pkgconfig
+  - /lib/systemd
+  - /mkspecs
+  - /modules
+  - /share/doc
+  - /share/man
+  - '*.prl'
+  - '*.la'
+  - '*.a'
+
+modules:
+  - name: tcl
+    buildsystem: simple
+    build-commands:
+      - cp -r ../tcl /app
+      - cd /app/tcl/unix; ./configure --prefix=/app --exec-prefix=/app && make && make install
+
+    sources:
+      - type: archive
+        url: https://chuanshaninfo.com/download/tcl8.6.14-src.tar.gz
```

Comment:
you should use the upstream source, not some random mirror.

Reply:
This project release at sourceforge, it seems can not be downloaded by build(due redirect). so I move it to my website.

Reply:
sourceforge works fine here. that's what matters.